### PR TITLE
Added more examples

### PR DESCRIPTION
**Pull request by abachma2**
Added a new notebook of examples using cymetric, materials_transactions.ipynb. I tried to incorporate use of the methods in cymetric. Open to changes to better use the methods, or adding/changing objective of each example.

### CODE REVIEW

File: examples/README.rst [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-    
-    $ python script_of_choice.py
-
-These scripts use the included database, ``recycle.sqlite``. The input file (``recycle.xml``) is also included for
+These example files use the included database, ``recycle.sqlite``. The input file (``recycle.xml``) is also included for
 your convenience. 
+
+The files ``agent_entryexit.py`` and ``agent_entryexit.ipynb`` walk through the same example, just with different 
+user interfaces. The example looks at the commissioning and decommissioning of reactor prototypes in the simulation. 
+The other example, ``materials_transactions.ipynb``, looks at the transfer of LWR fuel and the accumulation of waste in
```
Comment (katyhuff) [inline examples/README.rst:11]:
```suggestion
The other example, ``materials_transactions.ipynb``, plots the transfer of LWR fuel and the accumulation of waste in 
```

**Review by katyhuff — CHANGES_REQUESTED** [review]
This looks good to me. I made a minor comment. @bam241 the sqlite file is updated here, maybe just due to changes in cyclus since the last commit of recycle.sqlite? Unclear... something maybe to determine.

**Comment by bam241** [comment]
> This looks good to me. I made a minor comment. @bam241 the sqlite file is updated here, maybe just due to changes in cyclus since the last commit of recycle.sqlite? Unclear... something maybe to determine.

Yes this is possible, also cymetric writes by default the new table in the output file... which increases the size of the file...
so depending of what the previous implementer did... the size could vary....


@abachma2 looking at the notebooks, I would:
- add comment before each single line/blocks of code to explain precisely what they do.
- change the evaler definition line to `evaler = cym.Evaluator(db, write=False)` and explain that `=True` is a valid option as well. -- this is done with one not the other
- get reed of the `.reset_index()` when possible, I observed that in `cymetric` folks have been overusing those, often without gain...
- more the import after the output of the run, so people can relate to them without scrolling
- avoid to simple variable names (i.e. `l`, `cyg` etc...)
- maybe avoid aliases for `cymetric` import to help figure out which cymetric submodules one are using

Thank you for taking care of and updating this, this looks very nice!

**Comment by bam241** [comment]
just thinking, as we have a `recycle` repo for archetypes, maybe `recycle_cycle` would be a more appropriate name to avoid potential confusion...

**Comment by abachma2** [comment]
I think the revisions should address your comments @bam241. I kept the ``.reset_index()`` in ``agent_entryexit.ipynb`` because it wouldn't work without it. If you really think it needs to be removed, I can look for another way to accomplish the task. Let me know if there is anything I did not fully address

**Review by bam241 — CHANGES_REQUESTED** [review]
this is looking good, some few last comments:
- in `agent_entryexit.ipynb` you still have the cymetric import at the beginning they should be replaced by full import in box [3]
-  in `materials_transactions.ipynb` you still have some import in box [1] I'll group all of them together after the udge output of the cyclus run

**Comment by bam241** [comment]
(Sorry I started those comment last week, but got interrupted, and forgot to go back...)

**Review by bam241 — APPROVED** [review]
Looking good!

I'll let @katyhuff merge if she has no more comments.

**Review by katyhuff — APPROVED** [review]

**Comment by katyhuff** [comment]
Congrats @abachma2 !

**Pull request merged**